src/types/onyx/ReportNextStep.ts

```diff
@@ -0,0 +1,50 @@
+import {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+
+type Message = {
+    text: string;
+    type: string;
+    action: string;
+};
+
+type Button = {
+    text?: string;
+    tooltip?: string;
+    disabled?: boolean;
+    hidden?: boolean;
+    data?: unknown[];
+};
+
+type ReportNextStep = {
+    /** The message parts of the next step */
+    message: Message[];
```

Comment:
```suggestion
    message?: Message[];
```

According to the propTypes file.